environment-dev.yml

```diff
@@ -2,21 +2,24 @@ name: xdatasets
 channels:
   - conda-forge
 dependencies:
-  - python >=3.9,<3.13
+  - python >=3.10,<3.13
+  # Main packages
   - bottleneck >=1.3.1
-  - cftime >=1.4.1
+#  - cf-xarray >=0.6.1  # Was not included here, but appears in pyproject.toml
+  - cftime >=1.4.1  # Is it needed? It's not used in the code.
   - clisops >=0.9.2
   - dask >=2.6
   - dask-geopandas >=0.4.1
   - geopandas >=1.0
-  - intake <2.0.0  # Pinned until catalogs support v2.0 conventions. See: https://github.com/hydrologie/xdatasets/issues/82
+  - intake
   - intake-geopandas
   - intake-xarray >=0.6.1,<2.0.0
-  - ipykernel
-  - ipython >=8.5.0
+  - ipykernel  # This should probably not be in the environment-dev.yml
+  - ipython >=8.5.0  # This should probably not be in the environment-dev.yml
   - jsonpickle
-  - matplotlib-base >=3.5.0
+  - matplotlib-base >=3.5.0  # This should probably not be in the environment-dev.yml
```

Comment:
I would put `matplotlib` instead. `matplotlib-base` is matplotlib withoutb the Qt stack (better for conda packages).